test/src/template/template_spec.js

```diff
@@ -247,3 +248,68 @@ describe('attachView', function() {
     expect(template4Partial.templateObj[0].f[0].t).to.equal(types.ELEMENT);
   });
 });
+describe('registerWidget', function() {
+  var testWidget, noop;
```

Comment:
In Jasmine >= 2.0; `beforeEach`, `beforeAll`, `afterEach`, `afterAll`, and `it` share function context.

There is an alternative way to write this where defining variables in a higher scope can be skipped.

```js
describe('Some test concern', function() {
  beforeEach(function() {
    this.noop = function() {};
    this.testWidget = function() {};
  });
});
```

Whats nice about this is it prevents global test state and removes the need for cleaning this up in an `afterEach`.